### ISSUE TITLE
[Bug] Autotool uses wrong tool if high enchants

### ISSUE THREAD

**Issue by energy11a** [MeteorDevelopment/meteor-client#2097]
### Describe the bug

Autotool uses 32k stone hoe instead of unenchanted diamond_pickaxe to break cobblestone.

### Steps to reproduce

```bash
1.Enchant hoe with 32k efficiency, mending and 32k fortune
2.turn on autotool
3.get diamond_pickaxe
4.break cobblestone
```

### Meteor Version

1435

### Minecraft Version

1.18.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by Selbstverliebter** [comment]
How is this a bug

**Comment by energy11a** [comment]
It uses hoe to break cobblestone.

**Comment by Selbstverliebter** [comment]
Because its faster to break.

**Comment by energy11a** [comment]
Hoe does not break cobblestone faster.

**Comment by DesiCow** [comment]
Unable to reproduce on latest game and meteor version, it uses unenchanted diamond pickaxe over the 255 levels enchanted hoe as specified in bug report.

**Comment by Big-Iron-Cheems** [comment]
Can reproduce on the latest dev build, but the setup is a bit more complex than explained.

1. Have any tool (in his case a hoe) with efficiency higher than the pickaxe
2. Have Meteor Auto Tool on
3. Have Baritone Auto Tool on
4. Execute `#mine stone`

I believe the two Auto Tool settings conflict with each other.
If Baritone's is used, it will work as expected. 
If Meteor's is used, it will prefer the hoe for mining grass/dirt (unintended behavior?).
Using both of them will cause it to lock in place, probably constantly switching the items in hand.

**Comment by Big-Iron-Cheems** [comment]
Fixed in #3234, if any related bugs arise report them.
